templates/web/bexley/waste/_more_services_sidebar.html

```diff
@@ -15,7 +15,7 @@ <h3>More services</h3>
       <li><a [% external_new_tab | safe %] href="[% c.uri_for_action('waste/request', [ property.id ]) %]?request_type=removal">Order removal of old bins</a></li>
     [% END %]
     [% IF property.has_boxes %]
-      <li><a [% external_new_tab | safe %] href="[% c.uri_for_action('waste/request', [ property.id ]) %]?request_type=delivery&container-Deliver-Box-lids-55L=1">Order lids for recycling boxes</a></li>
+      <li><a [% external_new_tab | safe %] href="[% c.uri_for_action('waste/request', [ property.id ]) %]?request_type=delivery&amp;container-Deliver-Box-lids-55L=1">Order lids for recycling boxes</a></li>
```

Comment:
An ampersand declares the start of a special character; if instead of `container` after it you'd had, dunno, `=delivery&hearts=1`, then the HTML output would actually be `delivery♥=1`. So & should be escaped in HTML output.

Reply:
I did not realise it would do something like that with e.g. 'heart'. I'll have to try to remember that.